redash/handlers/organization.py

```diff
@@ -10,7 +10,7 @@
 @login_required
 def organization_status(org_slug=None):
     counters = {
-        'users': models.User.all_not_disabled(current_org).count(),
+        'users': models.User.all(current_org).count(),
```

Comment:
Why?

Reply:
If you look at https://github.com/getredash/redash/pull/3026/commits/56cd0da386f458538fe7f01a473ced6d4e4fbdeb#diff-2eb9bad8b59774e58b2c9a664440905aR516 it shows that `User.all` was already doing that, so `User.all_not_disabled` was redundant.